overlay/mkcrate.nix

```
@@ -231,7 +228,7 @@ let
     installPhase = ''
       mkdir -p $out/lib
       cargo_links="$(remarshal -if toml -of json Cargo.original.toml | jq -r '.package.links | select(. != null)')"
-      install_crate ${host-triple}
+      install_crate2 ${host-triple}
```

Do you think we could somehow check the `rustc` or `cargo` versions and fall back to the old behavior on older versions of Rust? I see that `install_crate` has been kept around and not removed.

---

Perhaps something like this?

```suggestion
      MINOR_RUSTC_VERSION="$(rustc --version | cut -d . -f 2)"
      if [[ "${MINOR_RUSTC_VERSION}" -ge 43 ]]; then
          install_crate2 ${host-triple}
      else
          install_crate ${host-triple}
      fi
```

---

Done in c5f4107, albeit in a slightly different form.